workalendar/europe/ukraine.py

```diff
@@ -0,0 +1,72 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+from datetime import date
+from workalendar.core import WesternCalendar, OrthodoxMixin
+from ..registry_tools import iso_register
+
+
+@iso_register('UA')
+class Ukraine(OrthodoxMixin, WesternCalendar):
+    'Ukraine'
+
+    shift_sunday_holidays = True
+    shift_new_years_day = True
+
+    FIXED_HOLIDAYS = WesternCalendar.FIXED_HOLIDAYS + (
+        (3, 8, "International Women’s Day"),
+        (5, 1, "Labour Day"),
+        #  (5, 2, "Labour Day"),  # Conflicting Sources
+        (5, 9, "Victory Day"),
+    )
+
+    include_good_friday = True
+    include_easter_sunday = True
+    include_easter_monday = True
+    include_whit_monday = True
+
+    def get_variable_days(self, year):
+        days = super(Ukraine, self).get_variable_days(year)
+
+        # Orthodox Christmas holiday is moved when it falls over the week
+        orthodox_christmas = date(year, 1, 7)
+        if orthodox_christmas.weekday() in self.get_weekend_days():
+            days.append((
+                self.find_following_working_day(orthodox_christmas),
+                "Orthodox Christmas (postponed)"))
+        else:
+            days.append((orthodox_christmas, "Orthodox Christmas"))
+
+        # Constitution Day was celebrated for the first time in 1996
+        if year >= 1996:
+            constitution_day = date(year, 6, 28)
+            if constitution_day.weekday() in self.get_weekend_days():
+                days.append((
+                    self.find_following_working_day(constitution_day),
+                    "Constitution Day (postponed)"))
+            else:
+                days.append((constitution_day, "Constitution Day"))
+
+        # Independence Day was first celebrated in 1991 on the 16th of June
+        # After the official independence has been moved to the 24th of August
+        if year >= 1991:
```

Comment:
maybe instead of having a "if >=" just a "if == 1991" would do. then you can go on with a "elif year >= 1992"

Reply:
speaking of which, if you have a "if" here, you may want to add a test for the years before 1991 and an other the year 1991.

Reply:
Absolutely. Will do.

Reply:
I don't see any tests for the year 1991. They may have been lost in your torough discussion with @alexdashkov ;o)

Reply:
I didn't include it because it looked clear enough.
I will add it asap